src/pages/Register/Register.tsx

```diff
@@ -64,30 +71,97 @@ const StyledPasswordInput = styled(PasswordInput)`
   margin-top: 20px;
 `;
 
+const ErrorField = styled(Field)`
+  display: none;
+`;
+
+const StyledErrorMessage = styled(ErrorMessage)`
+  color: ${props => props.theme.colors.functional.error};
+`;
+
+const validationSchema = object({
```

Comment:
Please see my previous comments about usage of Yup.InferType.
[https://github.com/jquense/yup#typescript-support](https://github.com/jquense/yup#typescript-support)